modules/apps/web-experience/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/portlet/view.jsp

```diff
@@ -18,11 +18,13 @@
 
 <%
 String productMenuState = SessionClicks.get(request, ProductNavigationProductMenuWebKeys.PRODUCT_NAVIGATION_PRODUCT_MENU_STATE, "closed");
+
+Group group = themeDisplay.getScopeGroup();
 %>
 
 <div class="lfr-product-menu-sidebar" id="productMenuSidebar">
 	<h4 class="sidebar-header">
-		<a href="<%= themeDisplay.getURLPortal() %>">
+		<a href="<%= HtmlUtil.escape(group.getDisplayURL(themeDisplay, layout.isPrivateLayout())) %>">
```

Comment:
I already talked with Eudaldo about this, but for the record: this change modifies the behaviour of the URL, which was previously correct: taking the user to the Home URL as defined in Instance Settings (or the "guest" site if no URL was defined).